google/cloud/forseti/common/util/file_uploader.py

```diff
@@ -33,11 +35,16 @@ def upload_json(data, gcs_upload_path):
         data (dict): the data to upload
         gcs_upload_path (string): the GCS upload path.
     """
-    with tempfile.NamedTemporaryFile() as tmp_data:
-        tmp_data.write(parser.json_stringify(data))
-        tmp_data.flush()
-        storage_client = StorageClient()
-        storage_client.put_text_file(tmp_data.name, gcs_upload_path)
+    try:
+        with tempfile.NamedTemporaryFile() as tmp_data:
+            tmp_data.write(parser.json_stringify(data))
+            tmp_data.flush()
+            storage_client = StorageClient()
+            storage_client.put_text_file(tmp_data.name, gcs_upload_path)
+    except HttpError as e:
```

Comment:
To make sure this is really resilient, I think that we should consider a broad except here.

And if you use logger.exception, you can capture the stacktrace within the context of the exception handler.  Thus saving an extra logger.error line.

```
except Exception:
    logger.exception('Unable to upload json document to bucket %s:\n%s',
                                 gcs_upload_path, data)
```